gravitee-am-ui/src/app/components/sidenav/sidenav.component.ts

```diff
@@ -98,6 +128,10 @@ export class SidenavComponent implements OnInit, OnDestroy {
     return this.environments && this.environments.length > 0 && !this.router.url.startsWith('/settings');
   }
 
+  canDisplayLicence(): boolean {
+    return !this.router.url.startsWith('/settings') && this.authService.hasPermissions(['license_notification_read']);
```

Comment:
**question**: Why are you limiting the notification to the `/settings` path ? I can't see anything into the JIRA description about this. IMO notification should alaways be displayed. wdyt ?